docs/guides/qiskit-addons-mpf.ipynb

```diff
@@ -0,0 +1,117 @@
+{
```

Comment:
<p>The acronym definition in the text should be (MPFs) since you are using the plural beforehand.</p><p> </p><blockquote>a system of linear equations which provide</blockquote><p>Should be "provides" because the system is singular.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/2367/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>